setup.py

```diff
@@ -16,9 +16,11 @@
 
 f = open("./requirements.txt",'r')
 install_req=f.readlines()
```

Comment:
we discussed this here: https://github.com/oda-hub/dispatcher-plugin-antares/pull/8 and highlighted that `install_requires` which is [meant to serve a bit different purpose than requirements.txt](https://packaging.python.org/discussions/install-requires-vs-requirements/).

Reply:
Only really the things used by the bare dispatcher in operations, and only references to pypi. Maybe it's worth revising if all packages listed are actually used.
tests_require equivalent can contain things needed for testing dispatcher, as long as they are in pypi.
Then, master branch of some packages, as well as plugins (needed for the environment) can be installed over it, from requirements.txt . I guess.